src/super_gradients/training/losses/yolox_loss.py

```diff
@@ -112,9 +112,24 @@ class YoloXDetectionLoss(_Loss):
     :param use_l1:                  Controls the L_l1 Coef as discussed above (default=False).
     :param center_sampling_radius:  Sampling radius used for center sampling when creating the fg mask (default=2.5).
     :param iou_type:                Iou loss type, one of ["iou","giou"] (deafult="iou").
+    :param iou_weight:              Weight to apply to the iou loss term.
+    :param obj_weight:              Weight to apply to the obj loss term.
+    :param cls_weight:              Weight to apply to the cls loss term.
+    :param cls_pos_weight:          Class weights for the cls loss. Passed on to torch.nn.BCEWithLogitsLoss
     """
 
-    def __init__(self, strides: list, num_classes: int, use_l1: bool = False, center_sampling_radius: float = 2.5, iou_type: str = "iou"):
+    def __init__(
+        self,
+        strides: list,
```

Comment:
Type annotation should be `List[int]` from typing and not built-in `list`.

Reply:
fixed